### PR TITLE
[Santorini] [HW1] class Student

### PR DESCRIPTION
**Pull request by yegorovmax**
class Student added

### CODE REVIEW

File: app/src/main/java/com/example/kotlinplayground/Lesson1.kt [hunk 1]
```diff
@@ -151,4 +153,14 @@ fun testClasses() {
     println("Lambo speed: ${lambo.speed}")
 
 //    audiCar.speed = 300   ---  Error
+
+    println("----- END CAR ------")
+}
+
+fun testStudentExam() {
```
Comment (tagirf) [inline app/src/main/java/com/example/kotlinplayground/Lesson1.kt]: I don't think that we need to change something here for homework.

**Review by tagirf — CHANGES_REQUESTED** [review]

File: app/src/main/java/com/example/kotlinplayground/StudentNameSantorini.kt [hunk 2]
```diff
@@ -0,0 +1,59 @@
+package com.example.kotlinplayground
+
+import kotlin.random.Random
+
+class StudentNameSantorini(numberOfStudentsInGroup: Int = 100) {
```
Comment (tagirf) [inline app/src/main/java/com/example/kotlinplayground/StudentNameSantorini.kt]: Hi @yegorovmax Looks like overengineering. Can you please simplify this class and push one commit with "all properties" and second teammate push commit with "all methods". Btw, you can take a look at the other PR.

**Review by tagirf — CHANGES_REQUESTED** [review]

File: app/src/main/java/com/example/kotlinplayground/StudentNameSantorini.kt [hunk 3]
```diff
@@ -0,0 +1,60 @@
+package com.example.kotlinplayground
+
+import kotlin.random.Random
+
+class StudentNameSantorini() {
+    val numberOfStudentsInGroup: Int = 100
```
Comment (tagirf) [inline app/src/main/java/com/example/kotlinplayground/StudentNameSantorini.kt]: This values should be private and in the class constructor

Comment (yegorovmax) [reply app/src/main/java/com/example/kotlinplayground/StudentNameSantorini.kt]: @tagirf  please take a look one more time 
fixed

File: app/src/main/java/com/example/kotlinplayground/StudentNameSantorini.kt [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+    private val numberOfStudentsInGroup: Int = 100
+    private var yearMathExam: Boolean = false
+    private var yearChemistryExam: Boolean = false
+    private var yearLinguisticsExam: Boolean = false
+    private var randomInt = Random.nextInt(0,1)
+    
+
+    fun exam(): Boolean {
+        // if randomInt 0 - exam failed, id randomInt is >0 exam passed
+        randomInt = Random.nextInt(0,15)
```
Comment (tagirf) [inline app/src/main/java/com/example/kotlinplayground/StudentNameSantorini.kt]: nit: let's use Kotlin features here! ` randomInt = (0..15).random()`

File: app/src/main/java/com/example/kotlinplayground/StudentNameSantorini.kt [hunk 5]
```diff
@@ -0,0 +1,37 @@
+package com.example.kotlinplayground
+
+import kotlin.random.Random
```
Comment (tagirf) [inline app/src/main/java/com/example/kotlinplayground/StudentNameSantorini.kt]: dead import, pls remove

**Review by tagirf — APPROVED** [review]
Looks good!

**Review by engeniousio — APPROVED** [review]

**Pull request merged**